web/web/auth/static/create_account.js

```diff
@@ -42,8 +42,7 @@ class CreateAccount extends React.PureComponent {
     };
 
     componentDidUpdate() {
-        if (this.props.authProps.isUserLoggedIn) {
-            this.props.authProps.dispatch(action.resetUserLoggedIn());
+        if (this.props.authProps.userData) {
```

Comment:
`userData`'s default value and the value you set when the login fails is: `{}` which is a truthy value in JS. So even when `userData` is an empty object, we'll redirect the user to: `window.location.href = menuRoutes[0].path;`
I'm assuming this is not intended.

Reply:
Will re-look at this, thanks for the catch

Reply:
done